log-viewer-cli/bin/config.conf

```diff
@@ -24,19 +24,43 @@ logs = [
 //    path: ${HOME}"/my-app/logs/*.log"
 //    format: {
 //      type: LogbackLogFormat
+//      locale: en_US // Set  custom locale (optional)
 //      pattern: "%date{yyyy-MM-dd_HH:mm:ss.SSS} [%thread] %-5level %logger{35} - %X{pipelineId}%X{contentId}%msg%n"
 //    }
 //  }
 ]
 
 authentication.enabled = false // Enabling/disabling Basic HTTP authentication
 
-// List of users for basic authentication. Used when `authentication.enabled=true`
+authentication.ldap.enabled = true // Enabling/disabling LDAP authentication
```

Comment:
`authentication.ldap.enabled` should be `false` by default. I'll change it after merge.